config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -290,4 +290,9 @@ config CHIP_LAST_FABRIC_REMOVED_ACTION_DELAY
 	  an action chosen by the CHIP_LAST_FABRIC_REMOVED_ACTION option. This schedule will allow for
 	  avoiding race conditions before the device removes non-volatile data.
 
+config CHIP_PERSISTENT_SUBSCRIPTIONS
```

Comment:
Why is this in `Kconfig.features` and not `Kconfig`?

Reply:
I thought about it as well when reviewing but the config/Zephyr where this is placed is generic for more platforms now. But I'm fine with adding experimental flag there - anyway.

Reply:
It is the Matter feature and it is optional AFAIR, but I don't have a strong opinion on where it should be added. :)

Reply:
Do you think that it fits better to Kconfig?

Reply:
Ok, I see, indeed we may do something with that to make it more intuitive :)

Reply:
Cool, i misdunderstood, that we want to move it to `Zephyr` folder. Thanks, all makes sense.